src/node/convolver.rs

```diff
@@ -214,24 +282,33 @@ impl ConvolverNode {
             1.
         };
 
-        // Pad the response buffer with zeroes so its size is a power of 2, with 2 * 128 as min size
-        let length = buffer.length();
-        let padded_length = length.next_power_of_two().max(2 * RENDER_QUANTUM_SIZE);
-        let samples: Vec<_> = (0..number_of_channels)
-            .map(|_| {
-                let mut samples = vec![0.; padded_length];
-                samples[..length]
-                    .iter_mut()
-                    .zip(buffer.get_channel_data(0))
-                    .for_each(|(o, i)| *o = *i * scale);
-                samples
-            })
-            .collect();
-
-        let padded_buffer = AudioBuffer::from(samples, sample_rate);
-        let convolve = ConvolverRendererInner::new(padded_buffer);
-
-        self.registration.post_message(Some(convolve));
+        let mut convolvers = Vec::<FFTConvolver<f32>>::new();
+        // Size of the partition changes a lot the perf...
+        // - RENDER_QUANTUM_SIZE     -> 20x (compared to real-time)
+        // - RENDER_QUANTUM_SIZE * 8 -> 134x
+        let partition_size = RENDER_QUANTUM_SIZE * 8;
+
+        [0..buffer.number_of_channels()].iter().for_each(|_| {
```

Comment:
As clippy suggests, iterating over `[Range]` does not do what you would expect. Perhaps better just `for _ in 0..buffer.number_of_channels()` ?